packages/react-cosmos-component-playground/src/components/ComponentPlayground/__tests__/fixture-editor/controls.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { Loader } from 'react-cosmos-loader';
+import createStateProxy from 'react-cosmos-state-proxy';
+import selectedEditorFixture from '../../__fixtures__/selected-editor';
+import DragHandle from '../../../DragHandle';
+import ComponentPlayground from '../../';
+
+// Vars populated in beforeEach blocks
+let wrapper;
+
+describe('Fixture editor controls', () => {
+  // Fixture editor is already on so the button will untoggle it
+  const fixtureEditorUrl = '/?component=ComponentA&fixture=foo';
+
+  beforeEach(() => {
+    return new Promise(resolve => {
+      // Mount component in order for ref and lifecycle methods to be called
+      wrapper = mount(
+        <Loader
+          proxies={[createStateProxy]}
+          component={ComponentPlayground}
+          fixture={selectedEditorFixture}
+          onComponentRef={resolve}
+        />
+      );
+    });
+  });
+
+  it('should set untoggle URL to fixture editor button', () => {
+    expect(wrapper.find(`.header a[href="${fixtureEditorUrl}"]`).length).toBe(
+      1
+    );
```

Comment:
I think we can also use [toHaveLength()](https://facebook.github.io/jest/docs/expect.html#tohavelengthnumber) if you want something neater.